templates/element.tera

```diff
@@ -0,0 +1,54 @@
+---
+accent: lavender
+whiskers:
+  version: "2.3.0"
+  matrix:
+    - flavor
+  filename: "themes/Catppuccin-{{ flavor.identifier }}.json"
+---
+
+{%- set palette = flavor.colors -%}
+
+{
+  "name": "Catppuccin {{ flavor.name }}",
+  "is_dark": {{ flavor.dark }},
+  "colors": {
+    "accent-color": "#{{ palette[accent].hex }}",
+    "primary-color": "#{{ palette[accent].hex }}",
+    "warning-color": "#{{ palette.red.hex }}",
+    "alert": "#{{ palette.yellow.hex }}",
+    "sidebar-color": "#{{ palette.crust.hex }}",
+    "roomlist-background-color": "#{{ palette.mantle.hex }}",
+    "roomlist-text-color": "#{{ palette.text.hex }}",
+    "roomlist-text-secondary-color": "#{{ palette.base.hex }}",
```

Comment:
Do you mind fixing
https://github.com/catppuccin/element/issues/4

Reply:
Separate PR I think. This just aims to refactor the build system atm.

Reply:
@isabelroses do you want to make the follow up PR or shall I?

Reply:
I don't mind doing it

Reply:
Also might be worth getting rid of unnecessary `palette` usage now that v2.3.0 is out